### PR TITLE
feat(fetch): switch to cross-fetch instead of isomorphic-fetch

### PR DESCRIPTION
**Pull request by JAdshead**
## Description

Switching to [cross-fetch](https://github.com/lquixada/cross-fetch) instead of [isomorphic-fetch](https://github.com/matthew-andrews/isomorphic-fetch) as the later has not been maintained for 5 years.

## Motivation and Context

isomorphic-fetch as the later has not been maintained for 5 years and provides outdated versions of both node-fetch and WHATWG fetch

## How Has This Been Tested?

Unit and integration tests.

## Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation (adding or updating documentation)

## Checklist:

- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
- [ ] These changes should be applied to a maintenance branch.
- [ ] This change requires cross browser checks.
- [ ] Performance tests should be ran against the server prior to merging.
- [ ] This change impacts caching for client browsers.
- [ ] This change impacts HTTP headers.
- [ ] This change adds additional environment variable requirements for One App users.
- [ ] I have added the Apache 2.0 license header to any new files created.

## What is the Impact to Developers Using One App?

Provides a fetch that is closer to the spec.

### CODE REVIEW

**Comment by infoxicator** [comment]
@JAdshead Integration tests pass so looks like everything is working same as with `isomorphic-fetch` 😁, if I want to run some manual tests, is this the only dep I need? "@americanexpress/one-app-bundler": "^6.4.0",

File: __tests__/integration/one-app.spec.js [hunk 1]
```diff
@@ -876,8 +879,8 @@ describe('Tests that can run against either local Docker setup or remote One App
           }
         );
         expect(response.status).toBe(200);
-        expect(response.headers._headers).toHaveProperty('access-control-allow-origin');
-        expect(response.headers._headers['access-control-allow-origin']).toEqual(['test.example.com']);
+        expect(response.headers.raw()).toHaveProperty('access-control-allow-origin');
```
Comment (mtomcal) [inline __tests__/integration/one-app.spec.js:882]: Would this new API of `raw` affect other wrappers or tools?

**Comment by JAdshead** [comment]
> @JAdshead Integration tests pass so looks like everything is working same as with `isomorphic-fetch` 😁, if I want to run some manual tests, is this the only dep I need? "@americanexpress/one-app-bundler": "^6.4.0",

Yes you will need the latest bundler

File: __tests__/integration/one-app.spec.js (hunk 1 above)
Comment (JAdshead) [reply __tests__/integration/one-app.spec.js:882]: Should not, the versions of `node-fetch` and `whatwg-fetch` are much closer aligned. the `headers._headers` is a node-fetch@v1 only api.

**Comment by mtomcal** [comment]
This looks good and given how many Integration tests where we use `fetch` this gives me confidence this is a non breaking swapout.

**Pull request merged**